plugin/src/exporter.rs

```diff
@@ -23,14 +23,29 @@ struct ActivityData {
     phase_span: Option<Context>,
 }
 
-fn fields_key_value(fields: &Vec<Field>) -> Option<KeyValue> {
-    if fields.len() > 0 {
-        return Some(KeyValue::new(
-                "nix.fields",
-                fields.iter().map(|v| format!("{v}").into()).collect::<Vec<String>>().join(" ")
-        ))
-    } else {
-        return None
+impl ActivityRecord {
+    fn build_attributes(self) -> Vec<KeyValue> {
+        let mut attributes = vec![KeyValue::new("nix.activitykind", format!("{:?}", self.kind))];
+        if self.fields.len() > 0 {
+            match self.kind {
+                ActivityKind::Build => {
+                    attributes.extend([
+                        KeyValue::new("storepath", format!("{}", self.fields[0])),
```

Comment:
I think these should be namespaced like nix.* to match the convention for custom OTel attributes